src/PoCs/ready.tsx

```diff
@@ -0,0 +1,18 @@
+import { Split, SplitItem } from "@patternfly/react-core";
+import React, {VoidFunctionComponent } from "react";
+import CheckCircleIcon from '@patternfly/react-icons/dist/js/icons/check-circle-icon';
+import "./style.css";
+export const ReadyForUse: VoidFunctionComponent  = () => {
+    return (
+        <div>       
+            <Split hasGutter className="foo">
+                <SplitItem>
+                    <CheckCircleIcon className="fixIcon" color="#3E8635" />      
```

Comment:
It would be good to use a common naming pattern for class names that we define.  I realize that the classes needed here are minimal, but establishing a pattern that we always follow will help when we get to components that require more customization via CSS.

Here's an example of the [class name that PF uses for their alert component](https://github.com/patternfly/patternfly/blob/main/src/patternfly/components/Alert/alert.scss#L188): 
`.pf-c-alert__icon`

Following PF naming patterns, maybe something like `mas-c-status__icon` would work? 

And then `foo` could be changed to `mas-c-status`.

@riccardo-forina WDYT about using `mas-` as a class namespace?

Also, can colors be defined using PF color variables? Should color styles be moved to the CSS file? E.g. we could define the PF color variables for modifier classes like `mas-m-ready` and `mas-m-failed`

Reply:
> @riccardo-forina WDYT about using `mas-` as a class namespace?

Ah, it's a good idea, I have been using `appservices` so far. Your suggestion is shorter.
But I was thinking about making the class name not needed at all by adopting [CSS modules](https://css-tricks.com/css-modules-part-1-need/). We are not as good as UX at writing CSS nor we need to write that much CSS, so I think not having to deal with figuring out a class name by leveraging the automatic class names from CSS modules sounds like a good idea to me. WDYT?

Reply:
> Also, can colors be defined using PF color variables?

I think so? In Storybook we have access to everything that [gets defined from these CSSs](https://github.com/redhat-developer/app-services-ui-components/blob/main/.storybook/preview.js#L1-L9). I think we'll have similar declarations on console dot.

We need to figure this out for certain.

Reply:
> But I was thinking about making the class name not needed at all by adopting [CSS modules](https://css-tricks.com/css-modules-part-1-need/). We are not as good as UX at writing CSS nor we need to write that much CSS, so I think not having to deal with figuring out a class name by leveraging the automatic class names from CSS modules sounds like a good idea to me. WDYT?

I'm not familiar enough with how CSS modules work to know if this meets our needs. Ideally, whenever we extend PF components or layouts with custom CSS, we would use PF variables for all of our styles where they exist (e.g. color, font-size, spaces). This way if the PF visual theme is ever updated (e.g. the make one of the colors brighter, or modify relative font-sizes), then it's less likely to be a breaking change.

Reply:
In short what CSS modules give you is a name for your classes that's automatically generated at build time. You don't need to come up with names that don't conflict with other existing classes and it's easier to refer to the CSS from the JS code.

Inside a CSS module you can still do everything you do with normal CSS files, so you have access to all variables etc.

CSS modules are good for one-shot customizations that you don't need to make reusable. They would not make sense when creating a CSS framework like PatternFly.